ballerina/smtp_client_endpoint.bal

```diff
@@ -40,11 +40,9 @@ public isolated client class SmtpClient {
     # + email - An `email:Message` message, which needs to be sent to the recipient
     # + return - An `email:Error` if failed to send the message to the recipient or else `()`
     remote isolated function sendMessage(Message email) returns Error? {
-        if (email?.contentType == ()) {
-            email.contentType = "text/plain";
-        } else if (!self.containsType(email?.contentType, "text")) {
+        if email.contentType is string && !self.containsType(email?.contentType, "text") {
```

Comment:
Should send an error if `email.contentType is ()` right

Reply:
No, if the `content-type` is not there the module itself should be able to identify a content-type for the email (by using `body` or `htmlBody` fields). This logic is implemented is already implemented in Java level. Hence removed it from here.